.github/workflows/unittest.yml

```diff
@@ -0,0 +1,41 @@
+name: Unittest
+
+on: [push]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tf-version: [1.13.1, 1.14.0, 1.15.0-rc2, 2.0.0]
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: |
+          pip install tensorflow==${{matrix.tf-version}}
+          pip install -e .[test]
+      - name: Test with pytest
+        run: pytest .
+  coverage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+      - name: Install dependencies
+        run: |
+          pip install tensorflow==2.0.0
```

Comment:
The old unit testing matrix (from `azure-pipelines.yml`) has TensorFlow 1.13.1 testing on Python 3.6, while here TensorFlow 2.0.0 runs coverage on Python 3.6. Is that right? Why isn't coverage running on Python 3.7 like before? (If I'm reading it right on line 19 of `azure-pipelines.yml`)

Reply:
Ideally we would test all TF versions against both 3.6 and 3.7, but I think that's a bit too much. I run the coverage on Python 3.6 to have at least one safety net if we introduce Python 3.7 syntax by accident.